react/week5/components/SearchUser.js

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect, useState } from 'react';
+import User from './User';
+
+function SearchUser({ username }) {
```

Comment:
Would probably have made a seperate component called Users that just get the users this component fetches.